jenkins/ci.suse.de/templates/cloud-update-ci-trigger-template.yaml

```diff
@@ -0,0 +1,19 @@
+- job-template:
+    name: 'cloud-update-ci-{instance}-trigger'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H */3 * * *'
```

Comment:
I don't care that much how often it is deployed automatically, as long as it is several times a week.